flink-cdc-connect/flink-cdc-source-connectors/flink-cdc-base/src/main/java/org/apache/flink/cdc/connectors/base/relational/connection/JdbcConnectionPoolFactory.java

```diff
@@ -43,6 +45,7 @@ public HikariDataSource createPooledDataSource(JdbcSourceConfig sourceConfig) {
         config.setMinimumIdle(MINIMUM_POOL_SIZE);
         config.setMaximumPoolSize(sourceConfig.getConnectionPoolSize());
         config.setConnectionTimeout(sourceConfig.getConnectTimeout().toMillis());
+        DriverManager.getDrivers();
```

Comment:
Could you add a note for this?

Reply:
OK, I have added a description in this PR.